package.json

```
@@ -20,34 +21,34 @@
     "build": "gulp build",
     "lint": "resin-lint lib tests gulpfile.coffee",
     "test-node": "gulp test",
-    "test-browser": "karma start",
+    "test-browser": "mockttp -c karma start",
     "test": "npm run lint && npm run test-node && npm run test-browser",
-    "prepublish": "npm test && npm run build"
+    "prepack": "npm run build"
```

I think this should be
```suggestion
    "prepare": "npm run build"
```

---

We are using `prepack` on the SDK so I blindly went for `prepack`.
TBH the docs description was too large to seem exactly what I was thinking about.
Will update it.